arch/src/aarch64/fdt.rs

```diff
@@ -178,6 +178,40 @@ pub fn get_cache_number_of_sets(cache_level: CacheLevel) -> u32 {
     }
 }
 
```

Comment:
The commit message should be start with component name, something like: `arch: fix cache sharing issue for cache passthrough on aarch64`.

Also please use the following format to refer related PR: `See: #5505 `

Reply:
Thanks , I will fix these issues soon.